risc0/zkvm/Cargo.toml

```diff
@@ -23,6 +23,7 @@ protobuf-src = { version = "1.1", optional = true }
 anyhow = { version = "1.0", default-features = false }
 bytemuck = "1.12"
 cfg-if = "1.0"
+getrandom = { version = "0.2", features = ["custom"] }
```

Comment:
I think the `getrandom` crate has a `std` flag, might be worth double checking if we need to optionally pass that feature flag on/off depending on if the zkvm is has it. https://docs.rs/crate/getrandom/0.2.8/features#std

Reply:
I've been playing around with various combinations and I'm unable to get a failure at least on the `getrandom` crate. I've run into some things using rand on the `multitests` but i'm able to get `rand` to work on the starter template. I'll merge this but I'll keep the `std` flag in mind.. I feel like we have some clean up to do on that front...